config/routes.rb

```diff
@@ -4,4 +4,10 @@
   resources :categories, only: [:index, :show], param: :slug
   resources :cart_stickers, only: [:create, :index]
   resources :cart, only: [:index]
+  get '/login', to: "session#new"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.